### PR TITLE
Cloud debug verbiage updates

### PR DESCRIPTION
**Pull request by jaffrepaul**
This ended up less Cloud, and more generalized. 
Small copy edits, cleanup and a new screenshot to remove the CT beta banner.

### CODE REVIEW

File: docs/guides/core-concepts/cypress-app.mdx [hunk 1]
```diff
@@ -426,18 +428,26 @@ The image below shows that our application is displaying at `1000px` width,
   alt="Viewport Scaling"
 />
 
+:::note
+
 _Note: The right-hand side may also be used to display syntax errors in your
```
Comment (elylucas) [inline docs/guides/core-concepts/cypress-app.mdx:433]:
```suggestion
The right-hand side may also be used to display syntax errors in your
```

File: docs/guides/core-concepts/cypress-app.mdx [hunk 2]
```diff
@@ -426,18 +428,26 @@ The image below shows that our application is displaying at `1000px` width,
...
+:::
+
 <DocsImage
   src="/img/guides/core-concepts/cypress-app/aut-error-e2e.png"
   alt="Errors"
 />
 
+:::note
+
 _Note: Internally, the AUT renders within an iframe. This can sometimes cause
```
Comment (elylucas) [inline docs/guides/core-concepts/cypress-app.mdx:445]:
```suggestion
Internally, the AUT renders within an iframe. This can sometimes cause
```

File: docs/guides/core-concepts/cypress-app.mdx [hunk 3]
```diff
@@ -508,10 +518,14 @@ spec file that prevent the tests from running._
   alt="Cypress app showing error as application under test"
 />
 
-_Note: Internally, the CUT renders within an iframe. This can sometimes cause
+:::note
+
+_Note: Internally, the AUT renders within an iframe. This can sometimes cause
```
Comment (elylucas) [inline docs/guides/core-concepts/cypress-app.mdx:523]:
```suggestion
Internally, the AUT renders within an iframe. This can sometimes cause
```

File: docs/guides/core-concepts/cypress-app.mdx [hunk 4]
```diff
@@ -426,18 +428,26 @@ The image below shows that our application is displaying at `1000px` width,
   alt="Viewport Scaling"
 />
 
+:::note
+
 _Note: The right-hand side may also be used to display syntax errors in your
 test file that prevent the tests from running._
```
Comment (elylucas) [inline docs/guides/core-concepts/cypress-app.mdx:434]:
```suggestion
test file that prevent the tests from running.
```

File: docs/guides/core-concepts/cypress-app.mdx [hunk 5]
```diff
@@ -426,18 +428,26 @@ The image below shows that our application is displaying at `1000px` width,
...
 <DocsImage
   src="/img/guides/core-concepts/cypress-app/aut-error-e2e.png"
   alt="Errors"
 />
 
+:::note
+
 _Note: Internally, the AUT renders within an iframe. This can sometimes cause
 unexpected behaviors
 [explained here.](/api/commands/window#Cypress-uses-2-different-windows)_
```
Comment (elylucas) [inline docs/guides/core-concepts/cypress-app.mdx:447]:
```suggestion
[explained here.](/api/commands/window#Cypress-uses-2-different-windows)
```

File: docs/guides/core-concepts/cypress-app.mdx [hunk 6]
```diff
@@ -508,10 +518,14 @@ spec file that prevent the tests from running._
   alt="Cypress app showing error as application under test"
 />
 
-_Note: Internally, the CUT renders within an iframe. This can sometimes cause
+:::note
+
+_Note: Internally, the AUT renders within an iframe. This can sometimes cause
 unexpected behaviors
 [explained here.](/api/commands/window#Cypress-uses-2-different-windows)_
```
Comment (elylucas) [inline docs/guides/core-concepts/cypress-app.mdx:525]:
```suggestion
[explained here.](/api/commands/window#Cypress-uses-2-different-windows)
```

File: docs/guides/core-concepts/introduction-to-cypress.mdx [hunk 7]
```diff
@@ -16,21 +16,20 @@ sidebar_position: 10
 
 :::tip
 
+<!-- // prettier-ignore -->
```
Comment (elylucas) [inline docs/guides/core-concepts/introduction-to-cypress.mdx:19]: why was this needed?

**Review by elylucas — APPROVED** [review]
thought that having the "Note:" in the admonitions was redundant so made some minor requests to take those out.

File: docs/guides/core-concepts/introduction-to-cypress.mdx (hunk 7 above)
Comment (jaffrepaul) [reply docs/guides/core-concepts/introduction-to-cypress.mdx:19]: Prettier forces the <Icon /> & punctuation onto new lines which renders a big dead space with that bit of copy at the bottom of the admonition. I'm sure there's a smarter way to handle these, but for now this is sort of handling artifacts from the previous site where these Font Icons are wedged into copy blocks.

**Review by elylucas — APPROVED** [review]

**Review by elylucas — APPROVED** [review]
